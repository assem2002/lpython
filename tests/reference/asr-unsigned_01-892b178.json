{
    "basename": "asr-unsigned_01-892b178",
    "cmd": "lpython --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/errors/unsigned_01.py",
    "infile_hash": "c176f4ec65c0b5ca7c2c95b2e35d65f22b4cef342baad15c2dd5ef24",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "asr-unsigned_01-892b178.stderr",
    "stderr_hash": "f19389db60f2168bed6694b7a2fc796362a9e5df5a5f5c0b63ee9ab0",
    "returncode": 2
}