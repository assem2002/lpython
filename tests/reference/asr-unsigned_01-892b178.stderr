semantic error: Cannot cast negative value to unsigned integer 
 --> tests/errors/unsigned_01.py:3:10
  |
3 | i: u16 = u16(-5)
  |          ^^^^^^^ 


Note: Please report unclear or confusing messages as bugs at
https://github.com/lcompilers/lpython/issues.
